runtime/src/tasks/qthreads/tasks-qthreads.c

```diff
@@ -682,6 +682,44 @@ void chpl_task_init(void)
 
     if (verbosity >= 2) { chpl_qt_setenv("INFO", "1", 0); }
 
+    // Explicitly bind shepherd threads to cores/PUs when using the
+    // binders topology.
+    char* topo = getenv("CHPL_QTHREAD_TOPOLOGY");
```

Comment:
Note that `CHPL_QTHREAD_TOPOLOGY` is currently a build time option and doesn't have to be set at execution time. Is this just temporary and if so, what's the longer term plan?

Reply:
I think this code also logically belongs in `setupAffinity()`